packages/components/src/notice/README.md

```diff
@@ -16,24 +16,12 @@ A Notice displays a succinct message. It can also offer the user options, like v
 
 Use Notices to communicate things that are important but don’t necessarily require action — a user can keep using the product even if they don’t choose to act on a Notice. They are less interruptive than a Modal.
 
-### Anatomy
-
-![Diagram of a Notice component with numbered labels](https://make.wordpress.org/design/files/2019/03/Notice-Anatomy.png)
-
-1. Container (status indicated with color)
-2. Icon (optional)
-3. Message
-4. Dismiss icon (optional)
-
```

Comment:
I pulled this list and the screenshot because I found the inclusion of an icon to be confusing. There isn't actually any icon implementation in the component (unless I've just overlooked it somehow). While it's possible to include one using `JSX` elements in the `children` prop, including it in a screen shot and list of the component anatomy implied to me that an icon would be part of the API, which doesn't appear to be the case.